sql/sql-server/src/test/scala/org/apache/spark/sql/server/SparkPgServerTestUtils.scala

```diff
@@ -95,7 +95,7 @@ class SparkPgSQLServerTest(
   private val className = SQLServer.getClass.getCanonicalName.stripSuffix("$")
   private val logFileMask = s"starting $className, logging to "
   private val successStartLines = Set(
-    "PgProtocolService: Start running the SQL server",
+    "PgService: Start running the SQL server",
```

Comment:
`PgService`?

Reply:
This is a bit confusing, but when running these tests on my machine, the log contains the following message:

```
18/05/25 11:57:51 INFO PgService: Start running the SQL server (port=14428, workerThreads=4)
```

I'm not quite sure why that is, so I probably need to research this a bit more to understand why these differences exist when running locally vs. travis/your machine. Clearly the class is called `PgProtocolService`, so I'm not sure why the log ends up this way.